requirements.txt

```diff
@@ -43,3 +43,6 @@ onnxruntime-gpu; sys_platform != 'darwin'
 torchcrepe==0.0.20
 fastapi==0.88
 ffmpy==0.3.1
+#下面是使用go-realtime-gui.bat需要依赖的包
```

Comment:
There is no need to write it here because it is in requirements-win-for-realtime_vc_gui.txt!

Reply:
Exactly.